state/state_test.go

```diff
@@ -419,11 +421,16 @@ func TestConsensusParamsChangesSaveLoad(t *testing.T) {
 	}
 }
 
-func makeParams(blockBytes, blockGas, evidenceAge int64) types.ConsensusParams {
+func makeParams(
+	blockBytes, blockGas int64,
```

Comment:
Nit: is there a reason those are paired while other params get a separate line?

Reply:
Same type, similar semantic meaning (ie. block maximums)